extensions/mqtt/ConsumeMQTT.cpp

```diff
@@ -35,7 +35,7 @@ namespace nifi {
 namespace minifi {
 namespace processors {
 
-core::Property ConsumeMQTT::MaxQueueSize("Max Flow Segment Size", "Maximum flow content payload segment size for the MQTT record", "");
+core::Property ConsumeMQTT::MaxQueueSize("Max Queue Size", "Maximum receive queue size for the MQTT record", "");
```

Comment:
May not want to change a variable name. If someone is using this option it may break their configuration.

Reply:
we need to correct that because the variable is not right in the first place. also if user was using that option, it will default to the default value.

Reply:
Do we intend to follow semver?

Reply:
The previous version of the parameter configuration is wrong so that is no backward compatible.

Reply:
I see your point that the variable name doesn't equal the option name "Max Flow Segment Size" vs "Max Queue Size" , but if a user happens to be using "Max flow segment size" per the option we've changed the semantics of what they want. Does that make sense? We should have that option available to them until we can deprecate it. Further, "receive queue size" seems to not connote if the size is in the number of flow files or the number of bytes. That should be clarified.

Reply:
Fixed in latest commit